client/components/errorLabel/errorLabel.html

```diff
@@ -0,0 +1,3 @@
+<link rel="stylesheet" href="./errorLabel.css">
+
+<error-label id="label_error" class="error" >Error</error-label>
```

Comment:
This is invalid html tag. By this I mean such html tag does not exist in HTML at all.
Also id is not needed for it either.